homepagecontent.php

```diff
@@ -0,0 +1,49 @@
+<?php session_start();
+    include "books.php"; ?>
+<html>
+<title>Homepage</title>
+
+<head>
+    <<meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="stylesheet" type="text/css" href="aboutus.css">
+</head>
+<?php include 'homepage.php'; ?>
+
+<body>
+
+    <?php 
+    echo "<br><br><br><br><br>";
+    ini_set("display_errors", 1);
+    error_reporting(E_ALL);
```

Comment:
Remove this code, move it to credentials file.

Reply:
ok